api/views.py

```diff
@@ -268,6 +268,7 @@ def get(self, request, format=None):
         data["is_fiat_sent"] = order.is_fiat_sent
         data["is_disputed"] = order.is_disputed
         data["ur_nick"] = request.user.username
+        data["satoshis_now"] = Logics.satoshis_now(order)
```

Comment:
This will add the attribute `satoshis_now` to the responses of the endpoint `/api/order/`  but not to the responses of `/api/book/` . It should be added also somewhere around Line 854 for this attribute to be sent once the list of orders is requested in BookView( ) https://github.com/Reckless-Satoshi/robosats/blob/3eeff8c142d11e776b327fe0d9f2495d4acca86a/api/views.py#L854

Reply:
I see, makes sense, I think I went too fast instead of installing the full platform and test the back-end 😞